build.gradle

```diff
@@ -87,6 +90,20 @@ configure(projectsWithFlags('java')) {
 
         tasks.withType(JavaCompile) {
             options.errorprone.excludedPaths = '.*/gen-src/.*'
+            // Open up the internal JDK API so that ErrorProne can access it.
```

Comment:
I think Gradle errorprone plugin does this so worth using if not yet

Reply:
Just realized we're using an old version. Updated to the latest one.